string moveGroup_name
uint16 index_axis
float32 angle
---
bool is_success
string info
